scripts/webpack/browser.js

```diff
@@ -76,15 +73,17 @@ module.exports = ({ isLegacyJS }) => ({
         rules: [
             {
                 test: /\.[jt]sx?|mjs$/,
-                exclude: [
-                    {
-                        test: /node_modules/,
-                        exclude: [
-                            /@guardian\/(?!(automat-modules))/,
-                            /dynamic-import-polyfill/,
-                        ],
-                    },
-                ],
+                exclude: {
+                    and: [/node_modules/],
+                    not: [
+                        // Include all @guardian modules, except automat-modules
+                        /@guardian\/(?!(automat-modules))/,
+
+                        // Include the dynamic-import-polyfill
+                        /dynamic-import-polyfill/,
+                    ],
+                },
```

Comment:
Please look closely at this, I think it is equivalent in the new syntax but it's a bit weird still.